### PR TITLE
Mobile Controls docs updates for build

### PR DESCRIPTION
**Pull request by Jahnp**
#### Pull request checklist
**NOTE:** This is just a cleanup PR for #8964 to fix the diff issues we ran into there. All credit for the contents here goes to @bigbadcapers, and to @ecraig12345 for setting up the stubs. Also, apologies for the messy git history--I'll take my lumps for forgetting to rebase.

- [x] Added mobile control page markdown
- [x] Added references to fabric-cdn to use images hosted there

###### Microsoft Reviewers: [Open in CodeFlow](http://wpcp.azurewebsites.net/CodeFlowProtocolProxy2.php?pullrequest=https://github.com/OfficeDev/office-ui-fabric-react/pull/8965)

### CODE REVIEW

**Comment by bigbadcapers** [comment]
I approve this update.

File: apps/fabric-website/src/pages/Controls/PivotPage/docs/ios/PivotImplementation.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+
+### Control name
+
+`MSSegmentedControl`
+
+### Library
+
+[UI Fabric iOS](https://github.com/OfficeDev/ui-fabric-ios)
+
+[Sample Code](https://github.com/bigbadcapers/ui-fabric-ios/blob/master/OfficeUIFabric.Demo/OfficeUIFabric.Demo/Demos/MSSegmentedControlDemoController.swift)
```
Comment (ecraig12345) [inline apps/fabric-website/src/pages/Controls/PivotPage/docs/ios/PivotImplementation.md]: Official repo link?

File: apps/fabric-website/src/pages/Controls/ListCells/docs/ios/ListCellsImplementation.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+
+### Control name
+
+`MSTableViewCell`
+
+### Library
+
+[UI Fabric iOS](https://github.com/OfficeDev/ui-fabric-ios)
+
+[Sample Code](https://github.com/bigbadcapers/ui-fabric-ios/blob/master/OfficeUIFabric.Demo/OfficeUIFabric.Demo/Demos/MSTableViewCellDemoController.swift)
```
[the remark this answers is not in the document]
Comment (bigbadcapers) [reply apps/fabric-website/src/pages/Controls/ListCells/docs/ios/ListCellsImplementation.md]: Yes

File: apps/fabric-website/src/pages/Controls/ListCells/ListCellsPage.doc.ts [hunk 3]
```diff
@@ -0,0 +1,13 @@
+import { TFabricPlatformPageProps } from '../../../interfaces/Platforms';
```
Comment (ecraig12345) [inline apps/fabric-website/src/pages/Controls/ListCells/ListCellsPage.doc.ts]: Looks like I messed up the naming of this folder, and that's why the build failed--the folder should be `ListCellsPage`, not `ListCells`.

File: apps/fabric-website/src/pages/Controls/PivotPage/docs/ios/PivotImplementation.md (hunk 1 above)
Comment (bigbadcapers) [reply apps/fabric-website/src/pages/Controls/PivotPage/docs/ios/PivotImplementation.md]: Approved

File: apps/fabric-website/src/pages/Controls/TooltipPage/docs/android/TooltipOverview.md [hunk 4]
```diff
@@ -0,0 +1,3 @@
+Use tooltips to show small unobtrusive hints on top of your app's UI. These can appear as a result of user interaction, or triggered thoughtfully to assist the user in learning the details of a new feature they have shown interest in.
+
+<img src="https://static2.sharepointonline.com/files/fabric/fabric-website/images/controls/android/tooltip/tooltipexample.png" alt="Examples of Various List Cells" style="width: 50%;" />
```
[the remark this answers is not in the document]
Comment (bigbadcapers) [reply apps/fabric-website/src/pages/Controls/TooltipPage/docs/android/TooltipOverview.md]: Approved. My bad.

File: apps/fabric-website/src/pages/Controls/PersonaPage/docs/android/PersonaOverview.md [hunk 5]
```diff
@@ -1,3 +1,9 @@
...
 
-The PersonaList view builds upon these controls to show a performant list, like you would see in an address book, or a People Picker auto-complete experience.
+Persona controls display the profile picture and online status of the user. When a user does not have a profile picture, the user's initials sit on top of a background color. In the event that both of these options are unavailable, we use a generic icon called a "doughboy".
+
+Beside a profile picture, Persona controls can include text elements. The user's full name can show alongside other metadata, such as job title, company, and more.
+
+Persona controls are also available as a performant list view. The 'PersonaList' controls are perfect for listing many people. Examples of this may be an Address Book or a People Picker auto-complete experience.
+
+<img src="https://static2.sharepointonline.com/files/fabric/fabric-website/images/controls/android/persona/personalistview.png" alt="Person Avatars in Various Sizes" style="width: 50%;" />
```
[the remark this answers is not in the document]
Comment (bigbadcapers) [reply apps/fabric-website/src/pages/Controls/PersonaPage/docs/android/PersonaOverview.md]: I’m struggling to change this via iPhone, but you’re right. Something like “People List in Android” should be appropriate. We can push the site with this glitch and I can fix it later.

File: apps/fabric-website/src/pages/Controls/ListCells/ListCellsPage.doc.ts (hunk 3 above)
Comment (Jahnp) [reply apps/fabric-website/src/pages/Controls/ListCells/ListCellsPage.doc.ts]: Thanks! Fixing locally.

**Pull request merged**